pkg/events/core_amd64.go

```diff
@@ -902,25 +902,458 @@ const (
 	Sys32process_mrelease             ID = 448
 )
 
-func DefaultSyscallsToCheck() []string {
-	return []string{
-		"read",
-		"write",
-		"open",
-		"close",
-		"ioctl",
-		"socket",
-		"sendto",
-		"recvfrom",
-		"sendmsg",
-		"recvmsg",
-		"execve",
-		"kill",
-		"getdents",
-		"ptrace",
-		"getdents64",
-		"openat",
-		"bpf",
-		"execveat",
-	}
+const SyscallNotImplemented = "NOT_IMPLEMENTED"
+
+// Order matters
+var SyscallSymbolNames = [...]string{
```

Comment:
This could be a map[ID]string declaration I think. Opened to other ideas, but a slice (and matching the index of this slice visually) would be harder to maintain.